en/lesson-08.md

```diff
@@ -57,98 +80,71 @@ source.
-macros to add them, the first is `\hline` adding a line across the full width of
-a `tabular`, the second is `\cline` which can be used to draw lines covering
-only a specific range of columns.
-
-`\cline` takes a mandatory argument that specifies the columns affected. It has
-to be a range with a hyphen (`-`) as a separator between the first and the last
-column number. You can specify multiple `\cline`s in the same row. Both an
-`\hline` and a `\cline` have to be the first content of a new row (so directly
-following `\\` or another `\hline` or `\cline`).
-
+The issue is that the `l` type column typesets its contents in a single row at
+its natural width, meaning taking as much space as it takes, even if there is a
+page border in the way. To overcome this you can use the `p` column. This one
+typesets its contents as paragraphs with the width you specify as an argument
+and vertically aligns them at the top &ndash; which you'll want most of the
+time. Compare aboves outcome to the following:
 
+<!-- {% raw %} -->
 ```latex
 \documentclass{article}
 \usepackage{array}
 
 \begin{document}
-\begin{tabular}{lll}
-  \hline
-  \hline
-  Animal & Food  & Size   \\
-  \hline
-  dog    & meat  & medium \\
-  \cline{1-2}
-  horse  & hay   & large  \\
-  \cline{1-1}\cline{3-3}
-  frog   & flies & small  \\
-  \hline
-  \hline
+\begin{tabular}{cp{9cm}}
+  Animal & Description \\
+  dog    & The dog is a member of the genus Canis, which forms part of the
+           wolf-like canids, and is the most widely abundant terrestrial
+           carnivore. \\
+  cat    & The cat is a domestic species of small carnivorous mammal. It is the
+           only domesticated species in the family Felidae and is often referred
+           to as the domestic cat to distinguish it from the wild members of the
+           family. \\
 \end{tabular}
 \end{document}
 ```
+<!-- {% endraw %} -->
 
-
-## Shortcuts and other preamble content
-
-The list of possible types in the table preamble above only showed the column
-alignments available, but LaTeX has more things to offer in the preamble. What
-is still missing is a way to change the horizontal space between two columns to
-something arbitrary. This can be done with
-
-type        | description
----         | :--
-`@{decl}` | replace the space between two columns with `decl`
-`!{decl}` | add `decl` in the centre of the existing space
-
-In case you're wondering now, by default each column has a horizontal space of
-the length `\tabcolsep` padded on both sides, resulting in a total of
-`2\tabcolsep` between columns &ndash; one from each column &ndash; and a single
-`\tabcolsep` on both outer ends. If we want to put a colon between the first
-two columns and change the space between the latter two to `1cm` we could do so
-by using
+If you want to declare many columns of the same type you'll think this is
+cumbersome to do for each. Luckily you can have an easier life in that case by
```

Comment:
`to do for each` what? Perhaps `you'll think this is cumbersome when the table has many columns`